cockatrice/src/dlg_tip_of_the_day.cpp

```diff
@@ -0,0 +1,110 @@
+#include <QCheckBox>
+#include <QDebug>
+#include <QDialogButtonBox>
+#include <QGridLayout>
+#include <QLabel>
+#include <QPushButton>
+
+#include "dlg_tip_of_the_day.h"
+#include "settingscache.h"
+#include "tip_of_the_day.h"
+
+DlgTipOfTheDay::DlgTipOfTheDay(QWidget *parent) : QDialog(parent)
+{
+    successfulInit = false;
+    QString xmlPath = "theme:tips/tips_of_the_day.xml";
```

Comment:
How does this path work if we're building with portable mode?

Reply:
I think it doesn't matter whether it's portable or not, because the xml has been added as resource and those are compiled to binary and get put inside the executable itself.